electrumsv/wallet.py

```diff
@@ -242,6 +241,41 @@ def __init__(self, wallet: 'Wallet', row: AccountRow,
         # TODO(rt12) BACKLOG formalise porting/storage of invoice data extracted from storage data
         self.invoices = InvoiceStore({})
 
+    @lru_cache(maxsize=2**16)
```

Comment:
Isn't this caching the script and script bytes for every output?

Reply:
I have merged the two into one now.